### ISSUE TITLE
[Question]: NullPointerException after migration to Android 14

### ISSUE THREAD

**Issue by OskarGosk** [Pushwoosh/pushwoosh-android-sdk#188]
### Documentation

- [X] I've checked the guide, but I didn't find the information I needed there.

### Question

Hello!

We are migrating our app to Android 14, but the app crashes when starting.

```
java.util.concurrent.ConcurrentHashMap.putVal (ConcurrentHashMap.java:1011)
java.util.concurrent.ConcurrentHashMap.put (ConcurrentHashMap.java:1006)
com.pushwoosh.tags.TagsBundle$Builder.putString (Unknown Source:2)
com.pushwoosh.appevents.b.a (SourceFile:2)
com.pushwoosh.appevents.b.b (SourceFile:4)
com.pushwoosh.appevents.a.a (SourceFile:1)
android.os.Handler.handleCallback (Handler.java:958)
android.os.Handler.dispatchMessage (Handler.java:99)
android.os.Looper.loopOnce (Looper.java:230)
android.os.Looper.loop (Looper.java:319)
android.app.ActivityThread.main (ActivityThread.java:8919)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:578)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1103)
```

We are using  version : 6.7.10

**Comment by wfhm** [comment]
@OskarGosk checking it.

**Comment by wfhm** [comment]
@OskarGosk Could you please specify what particular device model you were using in your tests?

**Comment by OskarGosk** [comment]
@wfhm 
![Zrzut ekranu 2024-07-24 131022](https://github.com/user-attachments/assets/2c1965bf-be96-4b02-b50c-b644625937a7)

I use Redmi Note 12 Android 14.

**Comment by wfhm** [comment]
@OskarGosk I was unable to reproduce the issue on vanilla Android devices and Huawei and Xiaomi devices I have at hand now, but I can also see this issue in our analytics now with a similar device distribution. The fix will be released shortly.

**Comment by OskarGosk** [comment]
Thanks for the information. I can't wait for the fixed release. Could you tell me when you plan to release a fix version?

**Comment by wfhm** [comment]
@OskarGosk I hope to release it today, but Maven Central is unstable now after yesterday's maintenance, throws 5xx errors when I try to publish. I'll let you know as soon as the new version is available.

**Comment by wfhm** [comment]
@OskarGosk Should be fixed in 6.7.11. Please let us know if the issue persists.

**Comment by OskarGosk** [comment]
@wfhm
 After updating the library to version 6.7.11, this problem did not occur again.
 Thanks for help!
